### ISSUE TITLE
NumMedia Param Missing: TypeError: can't convert nil into Integer

### ISSUE THREAD

**Issue by dlinch** [hellostealth/stealth-twilio#20]
Hello,

I'm spinning up a stealth bot for fun, but I hit a snag almost immediately. I added the stealth-twilio gem, added my credentials to the services.yml. I'm using (Insomia)[https://insomnia.rest/] to replicate api calls, and when I hit the base url (http://RANDOM_NGROK_STRING.ngrok.io/incoming/twilio) I see the request from Twilio in my logs, but I hit a type error, specifically: `WARN: TypeError: can't convert nil into Integer`

My stack trace points it to this [line:](https://github.com/hellostealth/stealth-twilio/blob/master/lib/stealth/services/twilio/message_handler.rb#L36)
Which makes sense, as I don't see that param in the response body from Twilio.

So either
1. I have an incorrect setup with Twilio, in which case I think the documentation could give a small walkthrough of which product integrates best with Stealth; I haven't done a Twilio integration before and their product offerings are a little overwhelming, so maybe I chose the incorrect product to associate with my account?
1. There is a logical error in the code. It's rescuing from an ArgumentError, but the exception raised is a TypeError. (Argument errors)[https://ruby-doc.org/core-2.2.0/ArgumentError.html] are generally raised when passing the incorrect number of arguments to functions, so I think the solution here is to change the code to:
```
 attachment_count = 0
begin
  attachment_count = Integer(params['NumMedia'])
  rescue TypeError
 end
```

Better yet, calling params['NumMedia'].to_i will default a nil to 0.

I opened a PR (#19) with my proposed edits, and some minor cleanup that I saw along the way if you'll have it.

**Comment by mgomes** [comment]
@dlinch thanks for the PR! Yep you are right, we should be rescuing from `ArgumentError, TypeError` there. I'll review the PR in a few. Thanks again.

**Comment by dlinch** [comment]
@mgomes Let me know if you need any maintainers; I really like this project and would love to get into some open source!

**Comment by mgomes** [comment]
@dlinch merged! Please keep the PRs coming 😬

Fixed in #19

**Comment by dlinch** [comment]
Sweet, thanks!
